icons/.svgrrc.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path');
+
+module.exports = {
+  "icon": false,
+  "expandProps": "end",
+  "replaceAttrValues": {
+    "#000": "currentColor"
```

Comment:
This is curiosity, but what exactly does this line do and why? (I assume replaces something with the currentColor but I'm wondering why we need that replacement.

Reply:
When I export svg files from Figma they have fill colors (black). When implemented we'd like to be able to control the color via css if we need to. `fill="currentColor"` on an svg path will make the color match the css controlled text color.

Reply:
This config is having svgr find any of the black fills and replacing them with currentColor.